webull/endpoints.py

```diff
@@ -82,20 +82,29 @@ def login(self):
         return f'{self.base_user_url}/passport/login/v5/account'
 
     def get_mfa(self) :
-        return f'{self.base_userbroker_url}/passport/v2/verificationCode/send'
+        return f'{self.base_userfintech_url}/passport/v2/verificationCode/send'
 
     def check_mfa(self) :
         return f'{self.base_userfintech_url}/passport/v2/verificationCode/checkCode'
 
-    def get_security(self, username, account_type, region_code, event, time, url=0) :
+	def get_security(self, username, account_type, region_code, event, time, url=0) :
```

Comment:
Line indent is off.  It should be 4 spaces instead of 8.

https://github.com/tedchou12/webull/blob/18df9cf27b161c2ae726b09f43b85ec6a7e11e70/webull/endpoints.py#L90

Reply:
Thanks fixing this, the second one should have been next_security as you can now change your security question.